resources/multisite_drupal_standard.make

```diff
@@ -507,6 +507,10 @@ projects[og][patch][] = patches/og-og_field_access-bypass_field_access-5159.patc
 ; https://www.drupal.org/node/1880226
 projects[og][patch][] = https://www.drupal.org/files/issues/og-use_numeric_id_for_membership_etid-1880226-5.patch
 
+; NEXTEUROPA-14012 Adding membership from user profile is in pending status
+; https://www.drupal.org/node/2744405
+projects[og][patch][] = https://www.drupal.org/files/issues/og_default_state_pending_2744405_11650415-20.patch
```

Comment:
@imanoleguskiza @voidtek Thanks for your review.
Forget what I said in the ticket. I checked again and this patch still needs to be included.
It was fixed in OG 7.x-2.x (https://github.com/Gizra/og/pull/124) but after the OG 2.9 release which the multisite uses ATM.

Regards,
Hervé